index.html

```diff
@@ -174,5 +174,26 @@ <h2>Skills</h2>
   </div>
   <div class="line"></div>
   </aside>
+  <footer>
+    <div class="footer-heading">
+    <h2 class="footer-heading">Contact me</h2>
+    <p id="description-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  </div>
+  <div class="decription">
+    <form action="https://formspree.io/f/mbjwpoar">
```

Comment:
- [ ] Your contact form is looking great, well done! However, it's not working properly because the `method` attribute is missing.

![Captura de pantalla 2022-04-26 a la(s) 10 55 19 p  m](https://user-images.githubusercontent.com/83717285/165437766-820f6c2b-93d7-4df8-b7a9-0fef28bbec2d.png)

Let's fix it by adding the method.